static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I think the existing solution is more efficient. As a DOM element is added, there is zero computation required to update the CSSOM and make it to the render tree. Whereas with a style attribute the HTML tokenizer and CSSOM builder need to work more, especially as items are added after the `DOMContentLoaded` event.